pkg/kubelet/dockertools/exec.go

```diff
@@ -90,15 +88,22 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 		if stderr != nil {
 			command.Stderr = stderr
 		}
-
-		return command.Run()
+		if err := command.Start(); err != nil {
```

Comment:
`Cmd.Run` is the same as `Cmd.Start` + `Cmd.Wait`.

What we're doing here is a refactoring, replacing `Run` with `Start` + `Wait` (see the last line of this method) so that we can write the timeout logic only once, instead of twice, in each branch of the if statement.